Scripts/script-findIncidentsWithIndicator.yml

```diff
@@ -0,0 +1,40 @@
+commonfields:
+  id: findIncidentsWithIndicator
+  version: 37
+name: findIncidentsWithIndicator
+script: |-
+  var res = executeCommand("getIncidents", {"query": "indicator.value:"+args.indicator});
+  var incidentsIDs=[];
+  var ec = {};
+  var md="#### Finding incidents with indicator: *" + args.indicator + "*\n";
+  if (res[0].Contents.data) {
+      var incidents = res[0].Contents.data;
+      for (var i=0;i<incidents.length;i++) {
+          if (!args.currentIncidentId || (args.currentIncidentId && args.currentIncidentId != incidents[i].id)) {
+              incidentsIDs.push(incidents[i].id)
+          }
+      }
+      md += "Found incidents IDs: " + incidentsIDs;
+      ec = {"IncidentsWithIndicatoer":[{"Indicator:": args.indicator,"incidentIDs":incidentsIDs}]};
+  } else {
+      md += "No incidents found"
+  }
+  return {Type: entryTypes.note, Contents: res, ContentsFormat: formats.json, HumanReadable: md, EntryContext: ec};
+type: javascript
+tags: []
+comment: Lookup incidents with specified indicator. Use currentIncidentId to omit
+  the existing incident from output
+enabled: true
+args:
+- name: indicator
+  required: true
+  default: true
+  description: Indicator to search for
+- name: currentIncidentId
```

Comment:
Why do you need this field ?
You are always running in a context of an incident